src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1065,6 +1066,10 @@ public void canCheckInLostAndPaidItem() {
     checkInFixture.checkInByBarcode(item);
 
     assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
+    Awaitility.await()
```

Comment:
Could a comment be added explaining why we expect given amount of events? if it is possible.